doc/community/contributing.md

```diff
@@ -0,0 +1,111 @@
+# How to Contribute
+
+Contributions are welcome, and they are greatly appreciated!
+Every little bit helps, and you always earn credits.
+
+You can contribute on the code side in many ways:
+
+* submit feedback,
+* add new features,
+* report bugs,
+* fix bugs,
+* implement a new cluster/cloud computation backend,
+* write documentation
+
+## Code
+
+### Linting and pre-commit
+
+For every code contribution you should run [pre-commit](https://pre-commit.com/index.html).
```

Comment:
Completely agree that it's much nicer to avoid pre-commit commits on the server but could it probably make sense to smoothen the tone a bit? Also, I think it's worth to mention that the pre-commit is run in an automated way. Adding an example (feel free to disregard),
```suggestion
For every code contribution we recommend to enable pre-commit running locally installing and activating [pre-commit](https://pre-commit.com/index.html).
```

Reply:
Revising slightly the text, making the sentence passive. It is also convenient for the user as having periodic diverging commit histories between local and github can be annoying